library/alloc/src/collections/btree/map.rs

```diff
@@ -811,7 +811,7 @@ impl<K: Ord, V> BTreeMap<K, V> {
     /// types that can be `==` without being identical. See the [module-level
     /// documentation] for more.
     ///
-    /// [module-level documentation]: crate::collections#insert-and-complex-keys
+    /// [module-level documentation]: index.html#insert-and-complex-keys
```

Comment:
What's going on here? Is this a regression?

Reply:
It was depending on the behavior of being resolved relative to `std` instead of relative to `alloc`: https://github.com/rust-lang/rust/pull/77253/commits/406584621a4e2d3752b2444c5bed2218447f0bfe

Reply:
I see.